packages/create-block/lib/templates.js

```diff
@@ -21,29 +21,48 @@ const prompts = require( './prompts' );
 const predefinedPluginTemplates = {
 	es5: {
 		defaultValues: {
-			slug: 'example-static-es5',
-			title: 'Example Static (ES5)',
+			slug: 'example-es5',
+			title: 'Example (ES5)',
 			description:
-				'Example static block scaffolded with Create Block tool – no build step required.',
+				'Example block scaffolded with Create Block tool – no build step required.',
 			dashicon: 'smiley',
 			wpScripts: false,
 			editorScript: 'file:./index.js',
 			editorStyle: 'file:./editor.css',
 			style: 'file:./style.css',
 		},
 		templatesPath: join( __dirname, 'templates', 'es5' ),
+		variants: {
+			static: {
+				slug: 'example-static-es5',
+				title: 'Example Static Block (ES5)',
+			},
```

Comment:
The values for the `static` variant could be set directly in the `defaultValues` object.

Reply:
I was pondering one more optimization from the perspective of the developer writing a scaffolding template. What if we assume that everything that is included in `defaultValues` is the default variant, so we never really need to pick it explicitly? If there are any variants, then we use the `variants` option, but we only provide non-default variants:

```js
variants: {
 	dynamic: {
 		slug: 'example-dynamic-block',
 		title: 'Example Dynamic Block',
 	},
 	anotherVariant: {},
},
```

That would generate 3 vars to use with the template:
- `isDefaultVariant`
- `isDynamicVariant`
- `isAnotherVariant`

If no variants were included, we wouldn't have any variables injected into templates.

This would also remove some checks like whether there is more than one variant provided because we would always have the default one and one or more in the `variants` field.

The only drawback is that we wouldn't have a way to pass the alternate name for the default variant so users won't see the radio option like:

- [x] static
- [ ] dynamic
- [ ] anotherVariant

Instead, it would look like

- [x] default
- [ ] dynamic
- [ ] anotherVariant

Reply:
I like the idea, perhaps we can iterate on it?

Reply:
I moved the default values from `static` to `defaultValues` in https://github.com/WordPress/gutenberg/pull/43481. I didn't refactor further. It's still valuable to have a way to name the default variant. In the future, we could also figure out how to pass some sort of description for the variant. At the moment it's `static` vs `dynamic` which might not be clear for someone starting with WP blocks.